Mac/MainWindow/Timeline/Cell/NSAttributedString+NetNewsWire.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  NSAttributedString+NetNewsWire.swift
+//  NetNewsWire
+//
+//  Created by Nate Weaver on 2020-04-07.
+//  Copyright © 2020 Ranchero Software. All rights reserved.
+//
+
+#if canImport(AppKit)
+import AppKit
+typealias Font = NSFont
+typealias FontDescriptor = NSFontDescriptor
+typealias Color = NSColor
+
+private let boldTrait = NSFontDescriptor.SymbolicTraits.bold
+private let italicTrait = NSFontDescriptor.SymbolicTraits.italic
+private let monoSpaceTrait = NSFontDescriptor.SymbolicTraits.monoSpace
+#else
+import UIKit
+typealias Font = UIFont
+typealias FontDescriptor = UIFontDescriptor
+typealias Color = UIColor
+
+private let boldTrait = UIFontDescriptor.SymbolicTraits.traitBold
+private let italicTrait = UIFontDescriptor.SymbolicTraits.traitItalic
+private let monoSpaceTrait = UIFontDescriptor.SymbolicTraits.traitMonoSpace
+#endif
+
+extension NSAttributedString {
+
```

Comment:
It looks like this file is common code, but is in the Mac folder.  Could we move this to Shared/Timeline?

Reply:
Oh, yes, good catch.

Reply:
I moved it into Shared/Extensions instead of Shared/Timeline.